pkg_name: "arm_move"

command_syntax_path: "/command/command_syntax.txt"
command_path: "/command/joint5_test.txt"

step: 0.01
sleep_time: 0.04
dimensions: [ 9, 16.5, 20, 11, 9.5, 11 ]
spline_distance_ratio: 2.5
gripper_actuation_time: 1.5

z_offcet: 0.0
link_radius: 1.5
